repos/keg-cli/src/tasks/docker/tunnel.js

```diff
@@ -0,0 +1,54 @@
+const ngrok = require('ngrok')
+const qrcode = require('qrcode-terminal')
+const { Logger } = require('KegLog')
+
+/**
+ * Create a public tunnel the the url (by default, to `kegdev.xyz`)
+ * @function
+ * @param {Object} args - arguments passed from the runTask method
+ * @param {string} args.command - Initial command being run
+ * @param {Array} args.params - arguments passed from the command line
+ *
+ * @returns {void}
+ */
+const tunnel = async args => {
+  const { params } = args
+  const { qr, source, port } = params
+
+  const ip = process.env.KEG_DOCKER_IP
+
+  Logger.header('Starting tunnel...', 'white')
+
+  const url = await ngrok.connect(source || port || ip)
+
+  Logger.spacedMsg('Source:', Logger.color('magenta', ip))
+  Logger.spacedMsg('Tunnel:', Logger.color('green', url))
+
+  // show a qr code on the terminal that the user can use to quickly access the tunnel on a mobile device
+  qr && qrcode.generate(url, { small: false })
+}
+
+module.exports = {
+  tunnel: {
+    name: 'tunnel',
+    alias: ['t', 'tl', 'ngrok' ],
+    action: tunnel,
+    description: 'Creates a public tunnel to the running docker container available at kegdev.xyz',
+    example: 'keg docker tunnel <options>',
+    options: {
+      qr:  {
```

Comment:
You can add a `default` key / value here for options and I would set `qr` to `true` by default
Looks like this => 
```js
      task: {
        description: `If true, "tunnel" will display a qr code that encodes the tunnel\'s public uri. This can make connecting a mobile device to the tunnel much easier.`,
        example: 'keg docker tunnel --qr false',
        default: true
      }
```

Reply:
👍  updated